src/utils/messages.ts

```diff
@@ -1,19 +1,37 @@
 import { MessageCenter as MC } from '@holoflows/kit/es'
-import { Person, Group } from '../database'
+import { Group, Person } from '../database'
 import Serialization from './type-transform/Serialization'
+import { PersonIdentifier, GroupIdentifier } from '../database/type'
 
-interface UIEvent {
-    closeActiveTab: undefined
-    settingsUpdated: undefined
+export interface PersonUpdateEvent {
+    readonly reason: 'update' | 'delete' | 'new'
+    readonly of: Person
 }
-interface KeyStoreEvent {
-    newGroup: Group
+interface MaskbookMessages {
+    /**
+     * Used to polyfill window.close in iOS and Android.
+     */
+    closeActiveTab: undefined
+    settingsCreated: string
     newPerson: Person
-    peopleChanged: undefined
     generateKeyPair: undefined
+    /**
+     * broadcast the settings updating
```

Comment:
Comments for this whole interface is... badly done. It should at least describe what the value, if provided, contains. The whole "broadcast" thing is also useless because ALL MESSAGES ARE BROADCAST. Just say "emitted on settings update / when settings are updated; contains instanceKey" or something, (Non-blocking; I will approve when someone else does.)